proto/ibc/tendermint/tendermint.proto

```diff
@@ -68,6 +68,26 @@ message ConsensusState {
   ];
 }
 
+// Evidence is a wrapper over tendermint's DuplicateVoteEvidence
+// that implements Evidence interface expected by ICS-02
+message Evidence {
```

Comment:
how do you differentiate between different forms of evidence?

Reply:
we currently only have one form of evidence for tendermint. Whenever we add more we can rename this to be more specific, but that is for another pr. Are we planning to support more than DuplicateVoteEvidence for 1.0? @cwgoes

Reply:
Yes; we want to support all of the evidence types which Tendermint supports.

Ideally we should just import the proto files from Tendermint directly, if they exist?

cc @cmwaters

Reply:
they are here: https://github.com/tendermint/tendermint/blob/master/proto/tendermint/types/evidence.proto.

I would recommend generating them yourself instead of importing them

Reply:
Yup, they are located [here](https://github.com/tendermint/tendermint/blob/master/proto/tendermint/types/evidence.proto) although they are still undergoing changes

Reply:
opened #7146 . @AdityaSripal do you think you could look into doing this? I'm not sure if this would require added logic in misbehaviour

Reply:
@cwgoes I don't see why we would want to support all the evidence types that Tendermint supports in core IBC (we will want to do this later for apps like cross-chain staking). Most are irrelevant for core IBC since they involve individual validators misbehaving.

What is core IBC supposed to do upon receiving Evidence of a single validator misbehaving (suppose a DoubleVote or Amnesia or Lunatic Evidence)?
Nothing. It is incumbent on the chain to punish this validator accordingly upon processing this evidence, but the IBC client should operate as normal as this is entirely within the concerns of the counterparty chain (It is part of the security model assumptions of Tendermint blockchains that individual validators may become malicious without worrying about the lite-client state).

IBC clients only need to do something when the security assumptions of the counterparty chain are violated. I.E. There is a light or full fork of the counterparty chain. This case is already handled in IBC's misbehaviour logic, though we could look into reusing the Evidence type Tendermint uses for this, namely `ConflictingHeadersEvidence`

Ref further dicussion here: https://github.com/cosmos/cosmos-sdk/issues/7146#issuecomment-679305265

Reply:
Ah I was confused by the comment above this message - https://github.com/cosmos/cosmos-sdk/issues/7146#issuecomment-679364939.